### PR TITLE
CORCI-812 release: Use raw_input() on Python 2

### PR DESCRIPTION
**Pull request by brianjmurrell**
Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

**Comment by brianjmurrell** [comment]
Functional test failures are both [DAOS-3945](https://jira.hpdd.intel.com/browse/DAOS-3945)

**Comment by brianjmurrell** [comment]
I really should have disabled hardware functional testing (probably all functional testing) on this PR.  :-(

File: SConstruct [hunk 1]
```diff
@@ -7,6 +7,11 @@ import time
 import errno
 from SCons.Script import BUILD_TARGETS
 
+try:
+    input = raw_input
```
[the remark this answers is not in the document]
Comment (jolivier23) [reply SConstruct]: should probably shut this warning up if this is intended

[the remark this answers is not in the document]
Comment (brianjmurrell) [reply SConstruct]: Done.

**Pull request merged**